src/history/hs_rec.c

```diff
@@ -861,8 +861,8 @@ __hs_delete_key_from_pos(WT_SESSION_IMPL *session, WT_CURSOR *hs_cursor, uint32_
          * inserting.
          */
         WT_WITHOUT_DHANDLE(session, ret = __wt_curhs_open(session, NULL, &insert_cursor));
-        F_SET(insert_cursor, WT_CURSTD_HS_READ_COMMITTED);
         WT_ERR(ret);
+        F_SET(insert_cursor, WT_CURSTD_HS_READ_COMMITTED);
```

Comment:
nitpick: would like if **insert_cursor** be called  **hs_cursor** or **hs_insert_cursor**

Reply:
hs_insert_cursor is better I think.

Reply:
I have fixed this function and another one in 68b9c98